File: app/src/main/java/com/example/applligent/nagoriengineering/model/ReminderModel.java
package com.example.applligent.nagoriengineering.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
@Entity
public class ReminderModel {

        @PrimaryKey(autoGenerate = true)
        public long id;
        public String user;
        public String message;
        public String time;
        public String date;
        public String sendingTime;
}

File: app/src/main/java/com/example/applligent/nagoriengineering/model/ReminderModel.kt
package com.example.applligent.nagoriengineering.model

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity
class ReminderModel {

    @PrimaryKey(autoGenerate = true)
    var id: Long = 0
    var user: String? = null
    var message = ""
    var time: String? = null
    var date: String? = null
    var sendingTime: String? = null
    var title: String? = null
    var userID: String? = null
}

